tasks/rsync-deploy.yml

```diff
@@ -15,13 +27,6 @@
     archive: yes
   delegate_to: "{{ inventory_hostname }}"
 
-# Ensure symlinks target paths is absent
-- name: ANSISTRANO | Ensure shared paths targets are absent
-  file:
-    state: absent
-    path: "{{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }}/{{ item }}"
-  with_items: "{{ ansistrano_shared_paths }}"
-
 # Symlinks shared paths
```

Comment:
That bit should be removed as well right? Please check #115 (although this was merged in your previous PR)